src/components/ha-menu-button.ts

```diff
@@ -98,7 +98,7 @@ class HaMenuButton extends LitElement {
       return;
     }
 
-    this.style.display = newNarrow || this._alwaysVisible ? "initial" : "none";
+    this.style.display = (newNarrow || this._alwaysVisible) && !this.hass.enableKioskMode ? "initial" : "none";
```

Comment:
This should be done on the user of this component

Reply:
Could you give me any ideas on how to do that? Still new to Polymer and this stuff.